docs/general/getting_started.md

```diff
@@ -15,6 +15,12 @@ And then execute:
 ```
 $ bundle
 ```
+### Available Commands
```

Comment:
Thing is, not all new users are rails users. Plus, most rails developers rarely use generators. I don't have a definitive opinion on this though, let's ask @joaomdmoura.